src/Genny.hpp

```diff
@@ -218,6 +221,10 @@ class Object {
         return add(std::make_unique<T>(name, args...));
     }
 
+    void set_generate(bool generate) { m_generate = generate; }
+
+    bool get_generate() const { return m_generate; }
+
```

Comment:
The reason I did this, is because not adding the type to genny makes it impossible for genny to use it.
For example the ProcessEvent calls sometimes need basic types such as FVector, if I force it to not be included it will just use a uint8_t and pad the structure, with this genny is aware of the type, its size etc but will not produce the files